src/test/kotlin/com/ampnet/mailservice/service/UserMailServiceTest.kt

```diff
@@ -28,6 +28,7 @@ class UserMailServiceTest : MailServiceTestBase() {
         suppose("Service sent the mail") {
             val request = MailConfirmationRequest.newBuilder()
                 .setEmail(testContext.receiverMail)
+                .setLanguage("invalid")
```

Comment:
This will be the most common value.
```suggestion
                .setLanguage("")
```